libsolidity/interface/CompilerStack.h

```diff
@@ -475,6 +476,7 @@ class CompilerStack: boost::noncopyable
 	bool m_generateIR = false;
 	bool m_generateEwasm = false;
 	std::map<std::string, util::h160> m_libraries;
+	std::map<ContractDefinition const*, std::unique_ptr<FunctionCallGraphBuilder::ContractCallGraph> const> m_contractCallGraphs;
```

Comment:
This region contains members that are all settings. I think right below `m_contracts` would be a better place. At best, it shouldn't be part of the CompilerStack at all...

Reply:
Maybe the ContractAnnotation would be a better fit?